tikv-client/src/main/java/com/pingcap/tikv/expression/LogicalBinaryExpression.java

```diff
@@ -102,4 +102,9 @@ public enum Type {
     OR,
     XOR
   }
+
+  @Override
+  public void setNewCollation() {
+    this.dataType.setCollation(left.dataType.getCollationCode());
+  }
```

Comment:
I think the method is the same as its' super class. why need to override it? 
If you just want to change collation in LogicalBinaryExpression, ComparisonBinaryExpression and StringRegExpression. I think super class should be an empty method?

Reply:
Thanks, I will make the super class an empty method.

Reply:
Thanks, I will make the super class an empty method.